osf/models/registrations.py

```diff
@@ -386,9 +386,10 @@ class DraftRegistration(ObjectIDMixin, BaseModel):
 
     datetime_initiated = NonNaiveDateTimeField(auto_now_add=True)
     datetime_updated = NonNaiveDateTimeField(auto_now=True)
+    deleted = NonNaiveDateTimeField(null=True, blank=True)
```

Comment:
Random thought: was there any discussion of making this `is_deleted` and adding a `deleted_date` to match node? Not that there is much consistency as files have a `deleted_on` and a `deleted_by_id`... 😐

Reply:
@alexschiller, consistency coming soon! See [OSF-8916](https://openscience.atlassian.net/browse/OSF-8916) (and Steve's comment on [OSF-8907](https://openscience.atlassian.net/browse/OSF-8907))